src/main/resources/public/js/getAndAddContracts.js

```diff
@@ -230,173 +250,122 @@ var projectContractsCount = -1;
             name: $("#name").text()
          }
 
-    
         function loadContracts() {
             $("#contracts").dataTable().fnDestroy();
             $("#contracts").dataTable({
-                //setup
-                serverSide:  true,
-                searching:   false,//searching not possible yet on server
-                ordering:    false,//same for ordering
-                //adapt DataTable request to Self-Paged API specification.
+                language: {
+                    loadingRecords: '<img src="/images/loading.svg" height="100">'
```

Comment:
@amihaiemil We could use the same aproach for the other invoice/task tables.
Basically 
```javascript
$("#table").dataTable({
     language: { loadingRecords: '<img src=...>' },
     ajax: function(_, callback){
         callRemote(...).then(remoteData => {
             callback({data: remoteData.map(toTableRow)})
         })
     }
});
```
Current loading for the other tables looks somewhat weird to me, table headers are pushed down by the svg:
![loading_invoices](https://user-images.githubusercontent.com/10284893/103292249-2109b980-49f6-11eb-9ee9-67f5786ec784.png)

Reply:
@criske yes, definitely. I will open a ticket for it.